installer/tag_release.sh

```diff
@@ -21,6 +25,14 @@ function git_show {
     git show -s --format='%h %s' $1
 }
 
+# Some machines only have `python3` in PATH, others have `python` - make an alias.
```

Comment:
would the `python3` binary ever be missing if the system has Python 3 installed? Plus this doesn't work if the system also has Python 2. I feel like we should just be using `python3` directly, since Python can be finicky around being aliased.

Reply:
Yeah, that's much simpler, thanks! Changed.